README.md

```diff
@@ -1 +1,10 @@
-# final-project
\ No newline at end of file
+# final-project
+
+
+Deployment 🔵
+
+Google cloud with Ubuntu 20.04 LTS VM.
+
+App deployed at: yonatanreed.mooo.com
```

Comment:
This link ( yonatanreed.mooo.com ) leads to 404  https://disk.yandex.ru/i/oRXp1WUUUXs5uA